admin/css/np_style.css

```diff
@@ -377,6 +362,15 @@ ul, ol {
     border-radius: 5px;
 }
 
+@media screen and (max-width: 1080px) {
```

Comment:
when the total of divs expands the container width, 1000px, it drops to the second row. 
Responsiveness might affect.